rdflib/term.py

```diff
@@ -1485,6 +1513,94 @@ def _parseBoolean(value: str) -> bool:
     return False
 
 
+def _well_formed_by_value(lexical: Union[str, bytes], value: Any) -> bool:
+    return value is not None
```

Comment:
This looked like an error on first glance, like it was intended to review `self._value` somehow.  After seeing how it's called, I see this is intended to be a fallback method.  Can it get a short documentation string summarizing this behavior?

Reply:
Added a docstring.

Reply:
Thanks!